rosbag2_transport/include/rosbag2_transport/rosbag2_transport.hpp

```diff
@@ -74,7 +74,8 @@ class Rosbag2Transport
    * \param play_options Options regarding the playback (e.g. queue size)
    */
   ROSBAG2_TRANSPORT_PUBLIC
-  void play(const StorageOptions & storage_options, const PlayOptions & play_options);
+  void play(
+    const StorageOptions & storage_options, const PlayOptions & play_options);
```

Comment:
Remove unrelated change
```suggestion
  void play(const StorageOptions & storage_options, const PlayOptions & play_options);
```

Reply:
done!